web-registry/src/lib/normalizers/normalizeEcocredits/normalizeEcocredits.constants.ts

```diff
@@ -0,0 +1,19 @@
+import { BatchInfo } from '@regen-network/api/lib/generated/regen/ecocredit/v1/query';
+
+import { ClassProjectInfo } from 'types/ledger/ecocredit';
+
+export const EMPTY_CREDIT_CLASS: ClassProjectInfo = {
+  className: '',
+  classId: '',
+  projectName: '',
+  projectLocation: '',
+};
+
+export const EMPTY_BATCH_INFO: BatchInfo = {
+  $type: 'regen.ecocredit.v1.BatchInfo',
```

Comment:
```suggestion
  $type: BatchInfo.$type,
```

this way we don't need to worry about changing the code here in case the msg type changes (eg bumping version)